### PR TITLE
#338 Fix template table layout for small screens

### PR DESCRIPTION
**Pull request by irinatelnova**

### CODE REVIEW

**Review by ensemenova — APPROVED** [review]

File: packages/ketcher-react/src/style/dialogs.less [hunk 1]
```diff
@@ -123,29 +123,40 @@ form[role='dialog'] {
       margin-left: 4px;
     }
   }
+  .table-group-wrap {
+    display: flex;
+    @media (max-width: @common-screen-tiny) {
+      flex-wrap: wrap;
+    }
```
Comment (TomasLight) [inline packages/ketcher-react/src/style/dialogs.less:130]: please fix according our discussion

File: packages/ketcher-react/src/style/dialogs.less [hunk 2]
```diff
@@ -123,29 +123,40 @@ form[role='dialog'] {
...
     width: 550px;
     height: 27em;
-    overflow: auto;
+    margin-top: 1em;
+    padding: 0;
+    overflow: hidden;
+    @media (max-width: @common-screen-tiny) {
+      width: auto;
+    }
+  }
```
Comment (TomasLight) [inline packages/ketcher-react/src/style/dialogs.less:150]: same

**Review by TomasLight — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/style/dialogs.less (hunk 1 above)
Comment (irinatelnova) [reply packages/ketcher-react/src/style/dialogs.less:134]: @TomasLight fixed

File: packages/ketcher-react/src/style/dialogs.less (hunk 2 above)
Comment (irinatelnova) [reply packages/ketcher-react/src/style/dialogs.less:152]: @TomasLight fixed

**Review by TomasLight — APPROVED** [review]

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
delete master features please

**Pull request closed without merging**